virttest/qemu_devices/qcontainer.py

```diff
@@ -593,9 +593,9 @@ def is_pci_device(self, device):
     def simple_hotplug(self, device, monitor, bus=None):
         """
         Function hotplug device to devices representation. If verification is
-        supported by hodplugged device and result of verification is True
+        supported by hotplugged device and result of verification is True
         then it calls set_clean. Otherwise it don't call set_clean because
-        devices representatio don't know if device is added correctly.
+        devices representation don't know if device is added correctly.
 
         :param device: Device which should be unplugged.
```

Comment:
should this be %s/unplugged/hotplugged ?

Reply:
@balamuruhans Yes, updated, thanks.